ddtrace/internal/remoteconfig/client.py

```diff
@@ -341,7 +347,7 @@ def _process_response(self, data):
         paths = {_.path for _ in payload.target_files}
         paths = paths.union({_["path"] for _ in self.cached_target_files})
 
-        if not set(payload.client_configs) <= paths:
+        if set(payload.client_configs) > paths:
```

Comment:
`!(A is a subset of B)` is not equivalent to `B is a proper subset of A`, so this is now a different check. Was the original one wrong?

Reply:
My bad, I didn't know `set_a() < set_b()`  == `set_a.issubset(set_b)` fixed in https://github.com/DataDog/dd-trace-py/pull/4988